src/Routes/Routes.tsx

```diff
@@ -105,7 +107,7 @@ export const Routes = memo(() => {
 
         if (isMobile && !state.isConnected) {
           const to = {
-            pathname: '/connect-wallet',
+            pathname: '/connect-mobile-wallet',
```

Comment:
q: I know this was part of the [original PR](https://github.com/shapeshift/web/pull/8446), but have we deployed this to an ephemeral env and confirmed mobile app splash is still happy?

Reply:
Confirmed it's working fine now on gome!